tools/flaky_test/run_process_xml.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+. tools/shell_utils.sh
+
+if [[ "${ENVOY_BUILD_ARCH}" == "aarch64" ]]; then
```

Comment:
You can just do
`export MULTIDICT_NO_EXTENSIONS=1 YARL_NO_EXTENSIONS=1` and then use python_venv as usual.

Reply:
oh. Wasn't sure if that is going to be propagated inside the virtual env. Will do this then!